jest.config.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
   collectCoverage: true,
 
   // An array of glob patterns indicating a set of files for which coverage information should be collected
-  collectCoverageFrom: ['./src/**/*.ts'],
+  collectCoverageFrom: ['./src/**/*.ts', '!./src/logging-utils.ts'],
```

Comment:
Hmm, this should be covered just via it being imported. Any reason to exclude it?